src/lando/conftest.py

```diff
@@ -98,18 +101,14 @@ def _patch(number=0):
 
 
 @pytest.fixture
-def git_setup_user():
-    return _git_setup_user
-
+def git_repo_seed() -> Path:
+    """
+    Return the path to a patch to set up a base git repo for tests.
 
-def _git_setup_user(repo_dir: Path):
-    """Configure the git user locally to repo_dir so as not to mess with the real user's configuration."""
-    subprocess.run(["git", "config", "user.name", "Py Test"], check=True, cwd=repo_dir)
-    subprocess.run(
-        ["git", "config", "user.email", "pytest@lando.example.net"],
-        check=True,
-        cwd=repo_dir,
-    )
+    The diff can  apply on an empty repo to create a known base for application
+    of other patches as part of the tests.
+    """
+    return Path(__file__).parent / "main" / "tests" / "data"
```

Comment:
This should be using `settings.BASE_DIR`.

Reply:
This is pre-existing code that  has been moved to the global conftest. I will update it.